src/Miew.js

```diff
@@ -1000,19 +1004,28 @@ Miew.prototype._onThemeChanged = (function() {
   };
 })();
 
-Miew.prototype._onBgColorChanged  = (function() {
-  return function() {
-    const gfx = this._gfx;
-    const color = settings.now.bg.color;
-    if (gfx) {
-      if (gfx.scene.fog) {
-        gfx.scene.fog.color.set(color);
-      }
-      gfx.renderer.setClearColor(color,  Number(!settings.now.bg.transparent));
+Miew.prototype._onBgColorChanged  = function() {
+  const gfx = this._gfx;
+  const color = chooseFogColor();
+  if (gfx) {
+    if (gfx.scene.fog) {
+      gfx.scene.fog.color.set(color);
     }
-    this._needRender = true;
-  };
-})();
+    gfx.renderer.setClearColor(settings.now.bg.color,  Number(!settings.now.bg.transparent));
+  }
+  this._needRender = true;
+};
+
+Miew.prototype._onFogColorChanged = function() {
+  const gfx = this._gfx;
+  const color = chooseFogColor();
+  if (gfx && gfx.scene.fog) {
+    gfx.scene.fog.color.set(color);
+  }
+  this._needRender = true;
+};
+
+Miew.prototype._onFogColorEnableChanged = Miew.prototype._onFogColorChanged;
```

Comment:
I think it's not necessary to have additional name for function processing fogcolor params changings. You can simply call _onFogColorChanged() in both cases for 'fogColor' and 'fogColorEnable'. Please, remove _onFogColorEnableChanged function.